Datasets/data-connect-dataset-teamschanneldetails.md

```diff
@@ -13,6 +13,14 @@ The TeamsChannelDetails_v0 dataset provides a list of Microsoft Teams channels.
 
 Channels are where the work actually gets done; where text, audio, and video conversations open to the whole team happen; where files are shared; and where tabs are added. TeamsChannelDetails dataset provides details of all the conversation channels created on Microsoft Teams for a particular tenant.
 
+NOTE: 
+
+- Currently, the MGDC platform ONLY supports extracting groups which have at least one valid user. Trying to extract groups not having any such valid user will result in pipelines failing with the error that none of the users requested were valid. Valid users must have the following:
```

Comment:
Based on the table, this should just have region filter:
![image](https://github.com/microsoftgraph/dataconnect-solutions/assets/129512615/7f3beab5-1d7a-4388-b4f6-6b26ae0cc0fb)

Reply:
Updated